truffle.js

```diff
@@ -1,7 +1,16 @@
+/* eslint-disable global-require */
 require("babel-register");
 require("babel-polyfill");
 const TestRPC = require("ethereumjs-testrpc");
 
+const providerUrl = "http://localhost:8545";
+const nanoPath = "44'/60'/0'/0`";
+
+const nanoProvider = () =>
```

Comment:
can you make providerUrl and nanoPath parameters to nanoProvider? and then instantiate properly later? then we can have many instances with different parameters

Reply:
You mean from the arguments that will be a tricky approach hence the idea mentioned below `_nano` we can set some flags and check for the parameter after the flag we will just need to make sure it doesn't activate truffle somehow.

Reply:
smth like this
```
const nanoProvider = (providerUrl, derivationPath, network) =>
  process.argv.filter(arg => arg === network).length > 0
    ? require("./nanoWeb3Provider").nanoWeb3Provider(providerUrl, derivationPath)
    : undefined;
```

Reply:
then we can create many instances like
`provider: nanoProvider("http://localhost:8545",  "44'/60'/0'/0", "nano")`